suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -167,6 +167,22 @@ export const getTitleForNetwork = (symbol: NetworkSymbol) => {
     }
 };
 
+export const getAccountTypePrefix = (path: string) => {
```

Comment:
I needed a different translation for different solana account types. The previous logic only recognized differences of the bip43 type, which is same for these solana paths.